### DISCUSSION TITLE
Suggestion: Python API for modding

### DISCUSSION THREAD

**Discussion by zelmor** [voxeltycoon/issues#447] [Modding]
I know there is a C# API in plan, but I wanted to ask if you are planning for a python interface too? I feel that pitching this idea with the [stackoverflow survey](https://insights.stackoverflow.com/survey/2019) is a cheap move, but I have no shame.

**Comment by andrewpey** [comment]
Currently, we have no such plans, but I can imagine a mod that allows you to do that.

**Comment by victorleblais** [comment]
Why Python and not javascript since it's way more popular according to the StackOverflow survey? :P

Like @andrewpey said, I guess it could be possible for people to make "interfaces" to expose the C# api to other languages.
